pkgs/development/python-modules/markdown-it-py/default.nix

```diff
@@ -27,19 +28,31 @@ buildPythonPackage rec {
     sha256 = "sha256-6UATJho3SuIbLktZtFcDrCTWIAh52E+n5adcgl49un0=";
   };
 
-  nativeBuildInputs = [
-    flit-core
+  patches = [
+    # Allow linkify-it-py v2.
+    # This only affects projects that depend on 'markdown-it-py[linkify]'
+    (fetchpatch {
+      # https://github.com/executablebooks/markdown-it-py/pull/218
+      url = "https://github.com/executablebooks/markdown-it-py/commit/96931bbc2afa4aafb321e24e64af20ec14a7187f.patch";
+      name = "hukkin-allow-linkify-v2.patch";
+      sha256 = "SqOtM0PXd8ZtSurqCqNEbJd9XoEAKl8B4D7kSC3Fq/Q=";
+    })
   ];
```

Comment:
Please use substituteInPlace to prevent apply errors in the future.

Reply:
It will also error?

Reply:
It will not error and also not complain if the surrounding lines change.